app/theme/index.tsx

```diff
@@ -92,6 +92,10 @@ export const config = defineConfig({
         orange: { ...defaultColors.orange, 500: { value: "#FF9916" } },
         yellow: { ...defaultColors.yellow, 500: { value: "#FFD80B" } },
         green: { ...defaultColors.green, 500: { value: "#00A651" } },
+        lime: {
+          100: { value: "#F7FBD9" },
+          400: { value: "#E3F37F" },
+        },
```

Comment:
@01painadam added lime as the "secondary" color instead; we can use that throughout as a colorPalette.